### DISCUSSION TITLE
blog/2022/220409-cloudflare-workers-arcaea-update/

### DISCUSSION THREAD

**Discussion by giscus** [zhanbao2000/blog#4] [Announcements]
# blog/2022/220409-cloudflare-workers-arcaea-update/

Arisa 的个人博客

https://blog.arisa.moe/blog/2022/220409-cloudflare-workers-arcaea-update/

**Comment by MBRjun** [comment]
Workers 被墙时代考古

**Comment by Edward334** [comment]
怎么到处都是源批（孬
